### PR TITLE
Roadmap update

### PR DESCRIPTION
**Pull request by jaygala223**
Fixed a minor grammatical error in ROADMAP.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,6 +57,19 @@ pip to install directly from the master branch on github:
...
+To use `pyenv`, you can follow the installation instructions on the [official website](https://github.com/pyenv/pyenv#installation). Once installed, create a new virtual environment with your desired Python version and activate it:
+```
+$ pyenv virtualenv <your-python-version> keras-nlp
+$ pyenv activate keras-nlp
+```
+Then you can proceed by installing `keras-nlp` and its dependencies in the activated environment by pasting the following in your command line interpreter:
+```
+$ pip install keras-nlp
+```
+
```
Comment (abheesht17) [inline README.md:72]: Should be removed from this PR.

**Comment by jaygala223** [comment]
Hi @abheesht17, I removed [added conda warning in README.md](https://github.com/keras-team/keras-nlp/pull/800/commits/9f2ea4c03e5b11dba35eff79659c24087b71a0d2) from the PR.

**Comment by abheesht17** [comment]
@jaygala223, it looks like you deleted the README. Could you please restore it? Just remove the part where you added the conda warning. Thanks!

**Comment by mattdangerw** [comment]
Looks like everything is separate commits, so I'll just grab the correct commit out of the batch.

**Pull request merged**